src/WinRT.Runtime/ExceptionHelpers.cs

```diff
@@ -63,24 +63,27 @@ static unsafe class ExceptionHelpers
 
         private static bool Initialize()
         {
-            IntPtr winRTErrorModule = Platform.LoadLibraryExW("api-ms-win-core-winrt-error-l1-1-1.dll", IntPtr.Zero, (uint)DllImportSearchPath.System32);
-            if (winRTErrorModule != IntPtr.Zero)
+            fixed (char* moduleName = "api-ms-win-core-winrt-error-l1-1-1.dll")
             {
-                roOriginateLanguageException = (delegate* unmanaged[Stdcall]<int, IntPtr, IntPtr, int>)Platform.GetProcAddress(winRTErrorModule, "RoOriginateLanguageException");
-                roReportUnhandledError = (delegate* unmanaged[Stdcall]<IntPtr, int>)Platform.GetProcAddress(winRTErrorModule, "RoReportUnhandledError");
-            }
-            else
-            {
-                winRTErrorModule = Platform.LoadLibraryExW("api-ms-win-core-winrt-error-l1-1-0.dll", IntPtr.Zero, (uint)DllImportSearchPath.System32);
-            }
+                IntPtr winRTErrorModule = Platform.LoadLibraryExW((ushort*)moduleName, IntPtr.Zero, (uint)DllImportSearchPath.System32);
+                if (winRTErrorModule != IntPtr.Zero)
+                {
+                    roOriginateLanguageException = (delegate* unmanaged[Stdcall]<int, IntPtr, IntPtr, int>)Platform.GetProcAddress(winRTErrorModule, "RoOriginateLanguageException"u8);
+                    roReportUnhandledError = (delegate* unmanaged[Stdcall]<IntPtr, int>)Platform.GetProcAddress(winRTErrorModule, "RoReportUnhandledError"u8);
+                }
+                else
+                {
+                    winRTErrorModule = Platform.LoadLibraryExW((ushort*)moduleName, IntPtr.Zero, (uint)DllImportSearchPath.System32);
```

Comment:
This doesn't seem correct, you're trying to load the same library again. The previous code was trying to load the `l1-1-1` .dll first, and falling back to `l1-1-0` otherwise. You're using the same .dll name in both cases instead 😅

Reply:
Whoops. Fixed now.